packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -556,253 +576,262 @@ export class StatsAnalyzer extends EventsScope {
+                `StatsAnalyzer:index#compareLastStatsResult --> ${simplifiedMediaType} level is 0 for the user`
+              );
+            }
           }
-        }
-
-        this.emitStartStopEvents(
-          mediaType,
-          previousStats.totalPacketsSent,
-          currentStats.totalPacketsSent,
-          true
-        );
-      }
 
-      if (this.meetingMediaStatus.expected.receiveAudio) {
-        // compare audio stats received
-        currentStats = this.statsResults[mediaType].recv;
-        previousStats = this.lastStatsResults[mediaType].recv;
-
-        if (
-          currentStats.totalPacketsReceived === previousStats.totalPacketsReceived ||
-          currentStats.totalPacketsReceived === 0
-        ) {
-          LoggerProxy.logger.info(
-            `StatsAnalyzer:index#compareLastStatsResult --> No ${mediaType} RTP packets received`
-          );
-        } else if (
-          currentStats.totalSamplesReceived === previousStats.totalSamplesReceived ||
-          currentStats.totalSamplesReceived === 0
-        ) {
-          LoggerProxy.logger.info(
-            `StatsAnalyzer:index#compareLastStatsResult --> No ${mediaType} samples received`
+          this.emitStartStopEvents(
+            simplifiedMediaType,
+            previousStats.totalPacketsSent,
+            currentStats.totalPacketsSent,
+            true
           );
         }
 
-        this.emitStartStopEvents(
-          mediaType,
-          previousStats.totalPacketsReceived,
-          currentStats.totalPacketsReceived,
-          false
-        );
-      }
+        if (this.meetingMediaStatus.expected.receiveAudio && mediaType.includes('audio-recv')) {
+          // compare audio stats received
+          currentStats = this.statsResults[mediaType].recv;
+          previousStats = this.lastStatsResults[mediaType].recv;
 
-      mediaType = STATS.VIDEO_CORRELATE;
-      if (this.meetingMediaStatus.expected.sendVideo) {
-        // compare video stats sent
-        currentStats = this.statsResults[mediaType].send;
-        previousStats = this.lastStatsResults[mediaType].send;
-
-        if (
-          currentStats.totalPacketsSent === previousStats.totalPacketsSent ||
-          currentStats.totalPacketsSent === 0
-        ) {
-          LoggerProxy.logger.info(
-            `StatsAnalyzer:index#compareLastStatsResult --> No ${mediaType} RTP packets sent`
-          );
-        } else {
           if (
-            currentStats.framesEncoded === previousStats.framesEncoded ||
-            currentStats.framesEncoded === 0
+            currentStats.totalPacketsReceived === previousStats.totalPacketsReceived ||
+            currentStats.totalPacketsReceived === 0
           ) {
             LoggerProxy.logger.info(
-              `StatsAnalyzer:index#compareLastStatsResult --> No ${mediaType} Frames Encoded`
+              `StatsAnalyzer:index#compareLastStatsResult --> No ${simplifiedMediaType} RTP packets received`
```

Comment:
I think it would be more useful for debugging to have the mediaType in the logs instead of the simplified one (applies to all the logs in `compareLastStatsResult()`)